src/Tooltip.js

```diff
@@ -34,7 +34,10 @@ const propTypes = {
     PropTypes.string,
     PropTypes.number
   ]),
-  innerRef: PropTypes.object,
+  innerRef: PropTypes.oneOfType([
+    PropTypes.func, 
+    PropTypes.string
```

Comment:
`PropTypes.object` as well, I was merely stating that it can be these 2 in addition to the one previously specified.